.github/move.yml

```diff
@@ -1,6 +1,7 @@
 # Configuration for move-issues - https://github.com/dessant/move-issues
```

Comment:
I don't think this is active is it? So I can drop the whole file.

Reply:
Drop it, we use native github move or ask the user to recreate issues in the correct repo.